### ISSUE TITLE
allow far Meadows 

### ISSUE THREAD

**Issue by 1010101110** [tueman/ValheimMods#6]
make a config option for meadows distance and check it here

https://github.com/tueman/ValheimMods/blob/188fb818ee13bd974a2185404560499d34133d61/WorldGenOptions/WorldGenOptions/Generator.cs#L78-L82

**Comment by tueman** [comment]
There has to be a default biome; if Meadows has limitations, something else has to be placed where it would have been. I could look into changing the way it works but in the meantime a good solution is to use the "switchers" config options to switch meadows with a biome that is far away (or made far away using config), like the plains. Of course if you do that you'll also spawn in plains.

**Comment by 1010101110** [comment]
yes this line just overrides its to make far meadows impossible, anything greater than 5000f blackforest is always used.

so to fix this change the 5000f to a variable.

magnitude > 9000f for example lets meadows spawn all the way to 9000f before the blackforest overrides them.

**Comment by tueman** [comment]
I think the best way to do this is to add the line of code you wrote and also add a "defaultBiome" variable that defaults to Meadows; I'll get this added.

**Comment by 1010101110** [comment]
i mean the existing code forces blackforests at anything over 5000f instead of 5000f hardcoded make it a config option

```cs 
      // this block of code forces blackforests to spawn at MEADOWSMAXDIST
     // default value is 5000f
      if (magnitude > MEADOWSMAXDIST+ num)
      {
          __result = WorldGenOptions.GenOptions.usingData.blackForestSwitch;
          return false;
      }

      //meadows is already default here
      __result = WorldGenOptions.GenOptions.usingData.meadowsSwitch;
      return false;
```
